app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::UsersController < ApiController
+  def index
+    @users = User.all.order(created_at: :desc)
+
+    render json: @users, status: :ok
+  end
+
+  def show
+    @user = User.find(params[:id])
+
+    render json: @user, status: :ok
+  end
+end
```

Comment:
## [OPTIONAL]

- [ ] As you have already the login authentication logic, I think it would be nice if you protect these routes(`before_action :authorize_request`) that are getting user info.